config.json

```
@@ -1,7 +1,7 @@
 {
   "definitions": {
     "npm": {
-      "default": "9.6.7+sha1.11902e3f00d4175bbd305e646ed82c2a14f1f588",
+      "default": "9.7.1+sha1.38819bf78a1847ad1bf7321ac593186da31f823e",
```

npm upgrade breaks the test, let's post-pone it
```suggestion
      "default": "9.6.7+sha1.11902e3f00d4175bbd305e646ed82c2a14f1f588",
```

---

Possibly related https://github.com/nodejs/corepack/pull/270